modules/ngiab_data_cli/__main__.py

```diff
@@ -45,28 +45,30 @@ def validate_input(args: argparse.Namespace) -> None:
 
     # always add or replace the prefix with cat if it is not a lat lon or gage
     if not args.latlon and not args.gage:
+        if not args.location:
+            raise ValueError("Please specify location with --location conus or --location hi, or use --latlon or --gage options.")
         input_feature = "cat-" + input_feature.split("-")[-1]
 
     if args.latlon and args.gage:
         raise ValueError("Cannot use both --latlon and --gage options at the same time.")
 
     if args.latlon:
-        feature_name = get_cat_id_from_lat_lon(input_feature)
+        feature_name, location = get_cat_id_from_lat_lon(input_feature)
         logging.info(f"Found {feature_name} from {input_feature}")
     elif args.gage:
-        feature_name = get_cat_from_gage_id(input_feature)
+        feature_name, location = get_cat_from_gage_id(input_feature)
         logging.info(f"Found {feature_name} from {input_feature}")
     else:
         feature_name = input_feature
-
+        location = args.location
     if args.output_name:
         output_folder = args.output_name
     elif args.gage:
         output_folder = input_feature
     else:
-        output_folder = feature_name
+        output_folder = f"{location}-{feature_name}" # there are duplicate cat-ids across different hydrofabrics
```

Comment:
```suggestion
        output_folder = feature_name
```
It's probably not too likely that someone's unlucky enough for this to be a problem. For now I think it would be better to preserve the existing naming and deal with this later with better overwrite protection #123